docs/conf.py

```diff
@@ -9,14 +9,11 @@
 
 import os
 import sys
-import inspect
 import shutil
 
 # -- Path setup --------------------------------------------------------------
 
-__location__ = os.path.join(
-    os.getcwd(), os.path.dirname(inspect.getfile(inspect.currentframe()))
-)
+__location__ = os.path.dirname(__file__)
```

Comment:
The previous version is more robust, but also more convoluted.
`__file__` should be enough for the majority of the usages of this particular file.

Reply:
hm... not so sure about that. You are changing the logic from absolute path to relative path, which might be problematic in some cases. How about:
```
__location__ = os.path.join(os.getcwd(), os.path.dirname(__file__)
```
as a compromise? And another thought. Since version 3.6 the `pathlib` module was already quite evolved. Maybe one can solve this problem already much simpler and nicer with pathlib?

Reply:
It seems that, [starting by Python 3.4](https://docs.python.org/3.4/whatsnew/3.4.html#other-language-changes), `__file__` is absolute.

I also did a quick test here to verify that:
```bash
$ echo "print(__file__)" > /tmp/test_python_file.py
$ python /tmp/test_python_file.py
__file__='/tmp/test_python_file.py'
```

I suppose if you want to use pathlib we could also do:
```python
from pathlib import Path

__location__ = Path(__file__).parent
```

Reply:
good to know that it's absolute now. I am also fine with `os.path.dirname(__file__)` as it might be even easier to read.